samples/net/openthread/coap_server/README.rst

```diff
@@ -1,13 +1,14 @@
 .. _coap_server_sample:
 
-openthread: coap_server
-##################
+OpenThread: CoAP Server
+#######################
 
-The openthread coap_server sample demonstrates interactions between nodes performing different Thread roles with the use of OpenThread and CoAP protocol.
+The OpenThread CoAP Server sample demonstrates interactions between nodes performing different Thread roles with the use of OpenThread and CoAP protocol.
 
 Overview
 ********
-
+You can use this sample application as a starting point to implement a CoAP application.
+This sample uses native OpenThread CoAP API for communication. For example usage of Zephyr CoAp API refer to :ref:`coap_client_sample`
```

Comment:
So this line is identical to the one in the client sample. One thing is that when I search for CoAP API in the client sample doc, I find this line and come back here, so it's a loop. This needs to be changed. If this refers to Zephyr, then link to Zephyr. If to the other sample, make sure there is more information about this API in the other sample.

Reply:
This line is exact opposite to the one on the client side:) Dependencies point to the proper documentation for given example, but I repeat that here too.